app/models/commontator/subscription.rb

```diff
@@ -7,7 +7,21 @@ class Subscription < ActiveRecord::Base
     validates_uniqueness_of :thread_id, :scope => [:subscriber_type, :subscriber_id]
 
     def self.comment_created(comment)
-      recipients = comment.thread.subscribers.reject{|s| s == comment.creator}
+      practice = Practice.find(comment.thread.commontable_id)
+      support_network_email = practice.support_network_email || nil
+      support_network_email_user = support_network_email.present? ? User.find_by(email: support_network_email.downcase) : nil
+      subscribers = comment.thread.subscribers
```

Comment:
Just so I am understanding the code correctly. `subscribers` can contain both `User` objects or an `email` as a `String`?

Reply:
Under normal circumstances, yes, `subscribers` are `User` objects. However, in the case in which a practice's `support_network_email` is not an existing `User` in the DB, I'm just adding the email to the array in order to include it in the `@recipient_emails` array in the `setup_variables` method from `subscriptions_mailer.rb`. Due to `commontator_thread.subscribers` not being an `ActiveRecord` collection, the changes do not persist after the request. You can check this in your rails console by getting the `commontator_thread.subscribers` both before and after the request, and they should be identical. I hope this was helpful. If not, we can meet to go over this :smiley:

Reply:
Thank you for the detailed explanation, Josh! Very helpful! :D